lib/kafo/hooking.rb

```diff
@@ -10,7 +10,8 @@ class Hooking
     # pre_commit - after validations or interactive wizard have completed, all parameter values are set but not yet stored in the answer file
     # pre - just before puppet is executed to converge system
     # post - just after puppet is executed to converge system
```

Comment:
For the sake of consistency the new type of hook should have a comment here explaining how it is used, similar to the others.